Blockzilla/OverlayView.swift

```diff
@@ -184,115 +168,98 @@ class OverlayView: UIView {
         button.setAttributedTitle(attributedString, for: .normal)
     }
     
-    func setSearchQuery(queryArray: [String], animated: Bool, hideFindInPage: Bool) {
-        searchQueryArray = queryArray
-        let query = queryArray[0].trimmingCharacters(in: .whitespaces)
-        let duration = animated ? UIConstants.layout.searchButtonAnimationDuration : 0
-
-        var showCopyButton = false
-
+    func setSearchQuery(suggestions: [String], hideFindInPage: Bool) {
+        searchQuery = suggestions[0]
+        searchSuggestions = searchQuery.isEmpty ? [] : suggestions
+        let isSearchSuggestionsPromptHidden = UserDefaults.standard.bool(forKey: SearchSuggestionsPromptView.respondedToSearchSuggestionsPrompt) || searchQuery.isEmpty
+        var isCopyButtonHidden = true
```

Comment:
nit: would recommend changing isCopyButtonHidden to copyButtonHidden (anywhere it occurs) for easier readability.